src/views/ViewAllExplorer.tsx

```diff
@@ -45,13 +63,13 @@ function ViewAllExplorer({
         />
       ) : (
         <FlatList
-          data={wallets || []}
+          data={_wallets}
           style={{
             maxHeight:
               Math.round(windowHeight * 0.6) - (StatusBar.currentHeight || 0),
           }}
           contentContainerStyle={styles.listContentContainer}
-          indicatorStyle={themeState.themeMode === 'dark' ? 'white' : 'black'}
+          indicatorStyle={themeMode === 'dark' ? 'white' : 'black'}
```

Comment:
Not sure how often you will be doing something like this, but you could introduce a hook like `useThemeVariable` or similar that returns correct on based on internal logic i.e. here could be `useThemeVariable(['white', 'black'])`